pkg/resolution/resolver/bundle/resolver_test.go

```diff
@@ -295,6 +296,16 @@ func TestResolve(t *testing.T) {
 			},
 			imageName:      "single-task",
 			expectedStatus: internal.CreateResolutionRequestStatusWithData(taskAsYAML),
+		}, {
```

Comment:
Thanks for the fix, wondering if we shall specify and have the mismatched test cases in the test case name?